### ISSUE TITLE
Change action test syntax

### ISSUE THREAD

**Issue by aklin** [aklin/minion#14]
Test cases at the moment are of the format `c.test.title` and `c.test.url` etc. The RH side is the regex to be performed.

This should change to a lisp-like RHS syntax to allow more flexibility, and untangle ourselves from defining a bunch of `c.test.*` rules.

Example:
```
c.test = (url "http://example.com") #Test string literal`
c.test = (url "x:*example*")             #Test regex
```

Experimentation should show which style is preferable.

**Comment by aklin** [comment]
Actually tests can be changed for assertions that can be ran on any element. This would be compatible with EAccess and shouldn't require separate code.

**Comment by aklin** [comment]
Closed in commit [86c3dfd](https://github.com/aklin/minion/commit/86c3dfd0f0a9b9bb7e695adaa638a029c22ed757)
